cfme/v2v/migrations.py

```diff
@@ -373,10 +372,18 @@ def plan_in_progress(self, plan_name):
                 # logger.info("For plan %s, current migrated VMs are %s out of total VMs %s",
                 #     migration_plan.name, view.progress_card.migrated_vms(migration_plan.name),
                 #     view.progress_card.total_vm_to_be_migrated(migration_plan.name))
-                self.logger.info("For plan %s, is plan in progress: %s,"
-                    "time elapsed for migration: %s",
-                    plan_name, is_plan_visible,
-                    plan_time_elapsed)
+                try:
+                    plan_time_elapsed = self.progress_card.get_clock(plan_name)
+                    new_msg = "time elapsed for migration: {time}".format(time=plan_time_elapsed)
+                except NoSuchElementException:
+                    new_msg = "playbook is executing.."
+                    pass
+                msg = "For plan {plan_name}, is plan in progress: {visibility}, {message}".format(
```

Comment:
**Optional**

No need to create new var here, its only used once in the log message.